### PR TITLE
Updated installation_windows_genric

### PR DESCRIPTION
**Pull request by KukkadapuSushma**

### CODE REVIEW

**Comment by daminisatya** [comment]
@KukkadapuSushma Is this tested thoroughly? since it is on windows

**Comment by KukkadapuSushma** [comment]
Yes I have and @samsruthi also had a look at that.

File: docs/installation_windows_generic.md [hunk 1]
```diff
@@ -25,10 +25,15 @@ Install in terminal:
...
-    > git checkout master
-    > ant jar
-    > java -jar dist/susiserver.jar
+    
+    	> ant jar
+    	> java -jar dist/susiserver.jar 
+	
+    (if on running above command gives error, try to change the branch either to "master" or "development" by following commands) :
     > git checkout development
+    > git checkout master
```
Comment (KukkadapuSushma) [inline docs/installation_windows_generic.md]: yeah that also works the same way! And yes, let's stick to one then! @samsruti

**Comment by daminisatya** [comment]
Thank you for the patch!

**Pull request merged**